services/web/client/source/class/qxapp/components/login/Form.js

```diff
@@ -12,18 +12,20 @@ qx.Class.define("qxapp.components.login.Form", {
   construct: function() {
     this.base(arguments);
 
-    // Items
-    let username = new qx.ui.form.TextField();
-    // TODO: add qx.util.Validate.checkEmail
     // TODO: add also login with user-id
+    // FIXME: WARNING add [DOM] Password field is not contained in a form: (More info: https://goo.gl/9p2vKq)
+
+    let username = new qx.ui.form.TextField();
     username.setRequired(true);
-    username.setPlaceholder("email");
-    this.add(username, "User", null, "user", null);
+    username.setPlaceholder("Your email address");
+    this.add(username, "", qx.util.Validate.email(), "username", null);
+    username.setTabIndex(1);
```

Comment:
username.set({
  required: true,
  placeholder: "Your email address",
  tabIndex: 1
})

Reply:
Is there any advantage besides compactness? E.g. atomic set?

Reply:
@oetiker